src/pages/guides/getting_started/code_playground_workflow.md

```diff
@@ -0,0 +1,193 @@
+2. Find the **Playground Sessions** section in the **Your add-ons** page
+3. Access your last session or create a new one with one click
+
+![Manage Your add-ons page](./img/playground-sessions.png)
+
+<InlineAlert slots="header, text1" variant="info"/>
+
+#### Accessing "Your add-ons" Page
+
+- **Without a document open:** Click the **Add-ons** button in the left rail, then click the **Add-on development** toggle in the top right
+- **With a document open:** Click the **Add-ons** button in the left rail, select the **Your add-ons** tab, then click the "Manage add-ons" link in the Add-on Testing section
+
+## Downloading Your Code
+
+### How to Download
+
+Downloading your code is a great way to save your work and continue working on it locally in your CLI.
+
+To download your code:
+
+1. Click the **...** button in the top right corner of the playground window
+2. Click the **Download** button
+3. This will download a zip file containing your code
+
+### Folder Structure
+
+The downloaded zip file will contain a folder with the following structure:
+
+- add-on folder
+- `script.js` file
+
+![Downloaded Folder Structure](./img/downloaded-folder-structure.png)
+
+### Running Downloaded Code
+
+You can run your add-on folder as a local add-on project in your CLI by following the steps in the [Quickstart Guide](../getting_started/quickstart.md#step-4-load-and-run-your-add-on). There is a readme file in the add-on folder that will guide you through the process as well.
+
+**Note:** You cannot run the `script.js` file alone.
+
+## Productivity Tips
+
+### Development Workflow
+
+1. **Start with Script Mode**: Use Script Mode to prototype and test your core functionality
+2. **Iterate Quickly**: Use keyboard shortcuts to save and run code frequently
+3. **Test Incrementally**: Run your code often to catch issues early
+4. **Transition to Add-on Mode**: Once your logic is working, move to Add-on Mode to build the UI
+5. **Download for Local Development**: Use the download feature to continue development locally
+
+### Code Organization
+
+- **Use Clear Naming**: Choose descriptive names for variables and functions
+- **Comment Your Code**: Add comments to explain complex logic
+- **Modular Approach**: Break down complex functionality into smaller functions
+- **Error Handling**: Always include proper error handling in your code
+
+### Testing Strategy
+
+- **Test with Different Documents**: Try your add-on with various document types and sizes
+- **Edge Cases**: Test with empty documents, single elements, and complex layouts
+- **Performance**: Monitor how your add-on performs with large documents
+- **User Experience**: Test the complete user flow from start to finish
+
+## Troubleshooting Common Issues
+
+### Code Not Running
+
+- Check for syntax errors in your code
+- Ensure you're using the correct mode for your use case
+- Verify that all required APIs are properly imported
+- Check the browser console for error messages
+
+### Session Not Saving
```

Comment:
Needs to be updated based on cloud persistence. There are certain limitations around cloud persistence. A session may not be backed up if the user has exceeded the session size limit or number of sessions allowed per user.